app/views/thank_you_mailer/thank_you_email.html.slim

```diff
@@ -0,0 +1,12 @@
+body
```

Comment:
`body` should not appear in the individual views - it should be part of the layout.